server/models/ViewRouteSentenceAndOffenceViewModel.ts

```diff
@@ -97,4 +97,35 @@ export default class ViewRouteSentenceAndOffenceViewModel {
     })
     return Object.keys(duplicates).map(key => [duplicates[key].caseSequence, duplicates[key].lineSequence])
   }
+
+  generateAdjustmentsArray() {
+    const adjustmentsArray = []
```

Comment:
Not a big fan of using array in the name. Maybe if you added a model it could be row or card as a suffix instead?

Reply:
Renamed to generateAdjustmentsRows and adjustmentsRows